build_engine_mapping.json

```diff
@@ -1,4 +1,6 @@
 {
   "vs-15-2017-win64-cxx17" : "Visual Studio 15 2017 Win64",
-  "vs-16-2019-win64-cxx17" : "Visual Studio 16 2019"
+  "vs-16-2019-win64-cxx17" : "Visual Studio 16 2019",
+  "windows-vs-16-2019-win64-cxx17" : "Visual Studio 16 2019"
```

Comment:
You just kind-of made just half the work here.

If we're touching that one anyway we should make the due dilligence to list all of the targets that actually need the generator properly set to `Visual Studio xx 20yy` . Espeacially as you potentially made another addition down below which isn't working (the cxx11 version defined `windows-vs-16-2019-win64.cmake`  namely....)